tests/target-browser/target_browser_suite_test.go

```diff
@@ -195,7 +195,7 @@ func deleteTarget(enableBrowsing bool) {
 }
 
 func runCmdBackupPlan(args []string) []targetbrowser.BackupPlan {
-	args = append(args, commonArgs...)
+	args = append(args, commonArgsWithJSONOutputFormat...)
```

Comment:
if `args` do not contain FlagOutputFormat then append FlagOutputFormat with common flags